dear_petition/petition/migrations/0019_auto_20200407_1720.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 2.2.10 on 2020-04-07 16:49
+
+from django.db import migrations
+
+from ..constants import (
+    SUPERIOR_COURT,
+    DISTRICT_COURT,
+    NOT_AVAILABLE,
+)
+
+from ..utils import make_datetime_aware, dt_obj_to_date
+
+
+def update_existing_ciprs_records(apps, schema_editor):
+    CIPRSRecord = apps.get_model("petition", "CIPRSRecord")
+    records = CIPRSRecord.objects.all()
+    for r in records:
+        refresh_record_from_data(r)
+
+
+def refresh_record_from_data(record):
+    """Updates model fields that depends on the data JSONField
+
+    The record exists, but the raw data (data - JSONFIELD) has
+    changed. Let's update the models that are extracting data
+    from this field.
+    """
+    record.file_no = record.data.get("General", {}).get("File No", "")
+    record.county = record.data.get("General", {}).get("County", "")
+    record.dob = record.data.get("Defendant", {}).get("Date of Birth/Estimated Age", "")
```

Comment:
```suggestion
    record.dob = record.data.get("Defendant", {}).get("Date of Birth/Estimated Age", None)
```
To match the method in the models.py file